### ISSUE TITLE
Pretraining code and initial instance pseudo-labels

### ISSUE THREAD

**Issue by egistific** [Kangningthu/ItS2CLR#7]
Hi,

Thank you for sharing your work. I wish to pretrain the feature extractor from scratch. Would it be possible to provide the pretraining code?
And it seems that features are extracted at the start of the finetuning phase (line 506 in https://github.com/Kangningthu/ItS2CLR/blob/main/train/trainer.py), but how are the initial instance pseudo-labels computed? args.pseudo_label_path  is required to build the datasets in run.py.

Thank you.
